server/api/themoviedb/index.ts

```diff
@@ -594,6 +607,24 @@ class TheMovieDb extends ExternalAPI {
       throw new Error(`[TMDB] Failed to fetch collection: ${e.message}`);
     }
   }
+
+  public async getRegions(): Promise<TmdbRegion[]> {
+    try {
+      if (!regions) {
+        const response = await this.axios.get<TmdbRegion[]>(
+          '/configuration/countries'
+        );
+
+        regions = sortBy(response.data, 'english_name');
+      }
+
+      return regions;
+    } catch (e) {
+      throw new Error(`[TMDB] Failed to fetch countries: ${e.message}`);
+    }
+  }
 }
 
+let regions: TmdbRegion[] | undefined;
```

Comment:
The first time the regions are retrieved from TMDB they will be stored here. I wanted to check if this makes sense, or if there is somewhere better?

Reply:
This... should work? We should define it above the class though. I wonder if we can just set a very long cache time for this request instead?

Reply:
Oh yeah, caching the request seems like it might be a more elegant solution.